### PR TITLE
Replace npm with yarn

### PR DESCRIPTION
**Pull request by mpetrunic**

### CODE REVIEW

**Comment by wemeetagain** [comment]
Could you also change the `.travis.yml` cache to `cache: yarn`?

**Comment by wemeetagain** [comment]
Looks like there's one last mention of npm in CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ We're trying to follow a few rules (the linter should catch most of these):
 >2. Do the changes in your fork
 >3. If you like the change and think the project could use it:
     * Be sure you have followed the code style for the project.
-    * run the linter within lodestar (npm run lint) and fix any issues if they arrise.
+    * run the linter within lodestar (yarn b lint) and fix any issues if they arrise.
```
Comment (wemeetagain) [inline CONTRIBUTING.md]: looks like a typo here

**Comment by GregTheGreek** [comment]
`yarn test` is failing on leveldb on my end

<details>
  <summary>Click For Logs</summary>

```
yarn run v1.15.2
$ yarn run check-types
$ tsc --noEmit
$ nyc -r lcov -e .ts -x "*.test.ts" mocha -r ./.babel-register "tests/**/*.test.ts" && nyc report

/Users/gregmarkou/tempClones/lodestar/node_modules/levelup/lib/levelup.js:74
    throw error
    ^
InitializationError: Must provide a location for the database
    at new LevelUP (/Users/gregmarkou/tempClones/lodestar/node_modules/levelup/lib/levelup.js:67:13)
    at LevelUP (/Users/gregmarkou/tempClones/lodestar/node_modules/levelup/lib/levelup.js:45:44)
</details>```

**Comment by mpetrunic** [comment]
> `yarn test` is failing on leveldb on my end
> 
> Click For Logs

I think it's because when I've generated yarn.lock (with first `yarn install`) levelup and leveldown versions changed and current initialization was wrong. Think you are running some old code because it's fixed with #157 (level wrapper is initializing levelup/down properly)

**Review by GregTheGreek — APPROVED** [review]
works on my end

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**